packages/asset/contracts/AssetReveal.sol

```diff
@@ -26,16 +26,18 @@ contract AssetReveal is
     mapping(address => mapping(uint256 => uint16)) revealIds;
     // signature nonces to prevent replay attacks
     mapping(address => uint32) public nonce;
```

Comment:
Name should change from nonce to signature nonce

Reply:
We named nonce for creator creator nonce as well right so I think naming it signature nonce would be a good idea

Reply:
Alright, since we will be using that in the AssetCreate as well I think you make a good point.